### ISSUE TITLE
Cannot fix line endings when open PHP tag is not on the first line

### ISSUE THREAD

**Issue by phlopsi** [squizlabs/PHP_CodeSniffer#807]
I have tested this with the current master branch https://github.com/squizlabs/PHP_CodeSniffer/commit/fe8c328f741aaebf583a3ce7ae26fd764b365e43.

When i try to fix the following PHP file with PSR-2 standard, phpcbf throws an error, when trying to convert the line endings. I marked the windows-style line endings with comments to visualize them.
The problem is the line ending preceding the `<?php`, otherwise it would work as intended.

test.php
```php
/* \r\n */
<?php/* \r\n */

```

```
$ phpcs --standard=PSR2 test.php

FILE: C:\[...]\test.php
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 2 | ERROR | [x] End of line character is invalid; expected "\n" but
   |       |     found "\r\n"
----------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------
```

```
$ phpcbf --no-patch --standard=PSR2 test.php
Changing into directory C:\[...]
Processing test.php [PHP => 2 tokens in 2 lines]... DONE in 4ms (1 fixable violations)
        => Fixing file: 1/1 violations remaining [made 50 passes]... ERROR in 57ms
Fixed 0 files
```

**Comment by gsherwood** [comment]
Thanks for reporting this. It is fixed now.

**Comment by raniesantos** [comment]
@gsherwood Hi I would like to know how this is fixed. I am using this with Visual Studio Code and I keep getting `End of line character is invalid; expected "\n" but found "\r\n"`

**Details:**

OS: Windows 7

Editor: Visual Studio Code `1.17.2`

phpcs extension used: https://github.com/ikappas/vscode-phpcs `v0.7.0`

`phpcs --version` PHP_CodeSniffer version 3.1.1 (stable) by Squiz (http://www.squiz.net)
